robottelo/ui/locators/model.py

```diff
@@ -253,11 +253,7 @@ def __repr__(self):
         if keys:
             msg.append("contains=%s" % str(keys))
         msg.append(">")
-        repr_text = u"|".join(msg)
-        if PY3:
-            return repr_text
-        else:
-            return repr_text.encode('utf-8')
+        return u"|".join(msg)
```

Comment:
this will not work in 2.7 when locator is unicode and can lead to exception as __repr__ in 2.7 expect always to return ascii str